python/cryptography/security/insecure-hash-algorithms.yaml

```diff
@@ -1,6 +1,9 @@
 rules:
   - id: insecure-hash-algorithm-md5
     pattern: cryptography.hazmat.primitives.hashes.MD5(...)
+    fix-regex:
+      regex: MD5
```

Comment:
This may have side-effects like cache-busting: MD5 is cheap/fast and unsuitable for cryptographic operations but is totally fine for things like cache membership.